# Use this file to specify parameters, that do not have default values.
# YAML Files can also be helpful if you want to save parameters for different use-cases.
# In any way: Default values should go into the .if file!
#
# To generate a yaml file, use "rosrun rosinterface_handler generate_yaml /path/to/KitafDetector.if"
#
# Use 'key: value' pairs, e.g.
# string: 'foo'
# integer: 1234
# float: 1234.5
# boolean: true
# vector: [1.0, 2.0, 3.4]
# map: {"a": "b", "c": "d"}

verbosity: info
color_image_topic: /kinect2/qhd/image_color_rect
depth_image_topic: /kinect2/qhd/image_depth_rect
image_publish_topic: /detections_image
direction_publish_topic: /direction_sign
service_name: /shut_down_detection
stop_publish_topic: /stop_sign
detect_score_threshold: 0.5
sign_score_threshold: 2.2
depth_threshold: 4.0
model_confidence: 0.95
publish_image: false
distance_percentile: 50
process_rate: 3